tsconfig.json

```diff
@@ -94,6 +94,6 @@
       ]
     },
     "transpileOnly": true,
-    "compiler": "ttypescript"
+    "compiler": "typescript"
```

Comment:
It's not a typo. See https://www.npmjs.com/package/ttypescript

Reply:
I see. Reverted.